connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/application/ReplE2ESuite.scala

```diff
@@ -1,154 +0,0 @@
-/*
```

Comment:
Because `shell: 'script -q -e -c "bash {0}"'` was deleted, this test will fail. Temporarily delete this file